aircraft/cost.py

```diff
@@ -0,0 +1,112 @@
+"""Aircraft cost models"""
+from gpkit.shortcuts import Var, Model
+import gpkit
+gpkit.disable_units()
```

Comment:
So, one problem with this is that I have a bunch of constants in the constraint functions that will need weird unit definitions because of the exponents.  I can change it so that the units balance out, but before I do I just want to make sure we're ok doing that.